### PR TITLE
[16.0][MIG] stock_landed_costs_analytic: Migration to 16.0

### PR DESCRIPTION
**Pull request by rov-adhoc**

### CODE REVIEW

**Review by jjscarafia — APPROVED** [review]
LGTM

File: stock_landed_costs_analytic/__manifest__.py [hunk 1]
```diff
@@ -7,11 +7,12 @@
         This module adds an analytic account and analytic tags on landed costs
         lines so that on landed costs validation account moves get analytic
         account and analytic tags values from landed costs lines.""",
-    "version": "13.0.1.0.0",
+    "version": "16.0.1.0.0",
     "license": "AGPL-3",
     "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/account-analytic",
-    "depends": ["stock_landed_costs"],
+    "depends": ["stock_landed_costs", "account_analytic_tag"],
```
Comment (pedrobaeza) [inline stock_landed_costs_analytic/__manifest__.py]: The module should work without this extra module with the base analytic that Odoo uses. Then, you can put an extra module for handling tags.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: stock_landed_costs_analytic/__manifest__.py (hunk 1 above)
Comment (rov-adhoc) [reply stock_landed_costs_analytic/__manifest__.py]: @pedrobaeza done

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump
/ocabot migration stock_landed_costs_analytic

**Pull request merged**